UI/Web/src/app/book-reader/_components/book-reader/book-reader.component.ts

```diff
@@ -1714,7 +1713,7 @@ export class BookReaderComponent implements OnInit, AfterViewInit, OnDestroy {
       case BookPageLayoutMode.Column1:
         return 1;
       case BookPageLayoutMode.Column2:
-        return calculationMethod === EpubPageCalculationMethod.Default ? 1 : 1.25;
+        return calculationMethod === EpubPageCalculationMethod.Default ? 2 : 2.5;
```

Comment:
The goal of the fix is to remove any need of having 2 different calculation modes.